tests/test_util/aiohttp.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, method, url, status, response):
 
     def match_request(self, method, url):
         """Test if response answers request."""
-        return method == self.method and url == self.url
+        return method == self.method and url.startswith(self.url)
```

Comment:
Non-async requests mock handled "startswith" type functionality with the old tests.